API_register/admin.py

```diff
@@ -1,3 +1,34 @@
 from django.contrib import admin
 
-# Register your models here.
+from API_register.models import Classes, Teacher, Announcements
+
+
+@admin.register(Classes)
+class MessagesAdmin(admin.ModelAdmin):
+    list_display = ['id', 'educator', 'name', 'description']
+
+
+@admin.register(Teacher)
+class MessagesAdmin(admin.ModelAdmin):
+    list_display = ['id', 'user']
+
+
+def content_display_thirty_signs(obj):
+    return str(obj.text)[0:30]
+
+
+content_display_thirty_signs.short_description = 'text'
+
+
+def deleted(model_admin, request, query_set):
+    query_set.update(deleted=True)
+
+
+deleted.short_description = "Ukryj element w widoku"
```

Comment:
deleted but, the description says hidden. Try to keep stuff standardized. I used to see this stuff named:
- Hidden
- Disabled
- Soft deleted
- Locked